src/vmm_resources.ml

```diff
@@ -105,7 +105,7 @@ let use_blocks t name vm active =
   | [] -> t
   | blocks ->
     let block_names =
-      List.map (fun (bd, dev) ->
+      List.map (fun (bd, dev, _sector_size) ->
```

Comment:
Maybe it could be interesting to be able to restrict sector sizes. For another pull request...

Reply:
I implemented the check in the client in https://github.com/roburio/albatross/pull/134/commits/48cfd63c297202ac6debaeb2e8ac292358e27000